pgsql/pc_pgsql.c

```diff
@@ -530,9 +530,10 @@ pc_patch_dimensional_serialize(const PCPATCH *patch_in)
 {
 	//  uint32_t size;
 	//  uint32_t pcid;
-	//  uint32_t compression;
+	//  uint16_t version;
```

Comment:
Version and compression should be swapped here and in all similar comments, to reflect the ordering in the declaration.